app.lua

```diff
@@ -126,8 +134,18 @@ function app:handle_404()
 end
 
 function app:handle_error(err, trace)
+    -- self.current_user is not available here.
+    local current_user = package.loaded.Users:find(self.session.username)
+    if current_user then
+        user_params = current_user:rollbar_params()
+    else
+        user_params = {}
+    end
     print(err)
     print(trace)
```

Comment:
Did you mean to keep this here? Where will it be printed if the server is running in the background?

Reply:
I did mean to leave them (they were already there..). I thought those were the statements that were logging stacktraces to the error.log. but that doesn't seem to be the case... So, I guess I should just remove them.